src/components/notebooks/card.tsx

```diff
@@ -1,12 +1,28 @@
-export default function Card({notebook} : { notebook?: Notebook }) {
-  // TODO: Fetch notebook 'Last Updated' when implemented on backend
-  const lastUpdated = '';
+import moment from 'moment';
+
+export default function Card({ notebook }: { notebook?: Notebook }) {
+  let editedText = '';
+
+  if (notebook) {
+    // If the notebook has been updated, use the updated time as the edited time
+    const notebookUpdatedAt = notebook.updated_at ? moment.utc(notebook.updated_at).local() : null;
+    const mostRecentPageUpdate = notebook.pages ? moment.max(notebook.pages.map(page => moment.utc(page.updated_at).local())) : null;
+
+    // If the notebook has no pages, use the notebook updated time else use the most recent page updated time
```

Comment:
I think we can simply use the notebook updated at. The backend will handle the logic of what makes it updated or not; in the backend it's planned that when a page is updated the notebook's `updated_at` will change.

Reply:
This is what I thought too, but during testing, the date did not change when I update a page, but did change when I update the notebook. It looks like the logic that you a re mentioning hasn’t been implemented, yet. Thus the reason why I handled it that way.